### PR TITLE
Edit Site: prevent inserter overscroll

### LINKED ISSUE #26428: Site Editor: opening the Inserter produces large overscroll
**Describe the bug**

There's a big white chunk of nothing at the bottom of the Site Editor with the Inserter open

**To reproduce**
1. Go to the experimental Site Editor
2. Click on the Inserter sidebar
3. Scroll down to the bottom of the Editor area
4. Notice that there's a huge white area of nothingness, a void if you will

Screen Shot 2020-10-23 at 11 53 26

**Expected behavior**
No void.

**Editor version (please complete the following information):**
- WordPress version: 5.5.1
- Gutenberg plugin version: 9.2.1

**Desktop (please complete the following information):**
 - OS: macOS 10.15.7
 - Browser: Chrome 86.0.4240.111

### PR DESCRIPTION
**Pull request by mattwiebe**
## Description
This just adds some `overflow: hidden` to `.interface-interface-skeleton__left-sidebar` to prevent it overflowing when open. It seems a bit too simple and may break something else, but since I believe we're already expecting overflows to be hidden (as in #26424), I hope there aren't any larger issues.

BONUS: the block inserter tip at the bottom is no longer cut off

fixes https://github.com/WordPress/gutenberg/issues/26428

## How has this been tested?
* Open Site Editor
* Open the Inserter sidebar
* See if there's a giant white space when you scroll down to the bottom of the Site Editor proper (not just the Inserter itself)
* See if the tooltip

### Before
Screen Shot 2020-10-23 at 16 34 10
^ Colorized green by me, normally white. Note the double scrollbars.

### After
Screen Shot 2020-10-23 at 16 30 28

## Types of changes

Bug fix

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/interface/src/components/interface-skeleton/style.scss [hunk 1]
```diff
@@ -110,6 +110,8 @@ html.interface-interface-skeleton__html-container {
 .interface-interface-skeleton__left-sidebar {
 	@include break-medium() {
 		border-right: $border-width solid $gray-200;
+		overflow: hidden; // prevent overscroll
+		padding-bottom: 24px; // compensate for breadcrumb footer
```
Comment (david-szabo97) [inline packages/interface/src/components/interface-skeleton/style.scss:114]: The height is actually 25px. 24px + 1px border top. 

How about using: `$button-size-small + $border-width` ?

**Review by david-szabo97 — CHANGES_REQUESTED** [review]
Tests well! I have one very minor request. See my comment abpve.

**Comment by Copons** [comment]
I've noticed that the editor's scrollbar gets cut off by the footer as well.
What about:
```scss
.interface-interface-skeleton__body {
	// ...
	@include break-medium() {
		padding-bottom: $button-size-small + $border-width;
	}
}

.interface-interface-skeleton__left-sidebar {
	@include break-medium() {
		// ...
		overflow: hidden;
	}
}
```

File: packages/interface/src/components/interface-skeleton/style.scss (hunk 1 above)
Comment (mattwiebe) [reply packages/interface/src/components/interface-skeleton/style.scss]: Ah yes, should have thought that we had these in variables, thanks!

**Comment by mattwiebe** [comment]
Thanks @Copons - better to deal with this on the body like you say.

> Also please make sure to test on the post editor as well, since these changes affect that too.

Yup, tested in there, everything seems unaffected.

**Review by david-szabo97 — APPROVED** [review]
Post editor ✅ Site editor ✅ Chrome ✅ Firefox ✅ Safari ✅ Ship it? ✅ 🚢

**Review by vindl — APPROVED** [review]

**Comment by talldan** [comment]
This was shipped in 5.6.1, so removing the backport label:
https://make.wordpress.org/core/2021/01/28/wordpress-5-6-1-rc1/

**Pull request merged**